features/prompt-history/request-id.mdx

```diff
@@ -33,6 +33,23 @@ response, pl_request_id = openai.Completion.create(
 )
 ```
 
+## Javascript
+
+To retrieve the `pl_request_id` using OpenAI in JavaScript (through `promptlayer.OpenAI`), set the argument `return_pl_id` to `true` in your function call. This will change the call to now return the OpenAI response and the `pl_request_id`. If you are using `stream: true`, then only the last `pl_request_id` will be the id; otherwise, it will be `undefined`.
+
+The same is true for Anthropic.
+        
+```js
```

Comment:
Should we add a Typescript example here as well? Just for that ignore line?

Reply:
@jzone3 there is a code snippet [here](https://github.com/MagnivOrg/prompt-layer-docs/blob/9b7f09dd4ea4709ebe3db3d25133dd2f90484c14/languages/javascript.mdx#L64) for TypeScript. I think that is sufficient enough.